AsyncDisplayKit/ASCellNode.m

```diff
@@ -27,6 +28,7 @@ - (instancetype)init
   // use UITableViewCell defaults
   _selectionStyle = UITableViewCellSelectionStyleDefault;
   self.clipsToBounds = YES;
+  _relayoutAnimation = UITableViewRowAnimationAutomatic;
```

Comment:
What's the purpose of having a `suggestedAnimation` param when this is always automatic?